framework/include/mesh/MooseMesh.h

```diff
@@ -932,7 +932,7 @@ class MooseMesh : public MooseObject, public Restartable, public PerfGraphInterf
    * @param subdomain_id The subdomain ID you want to get the boundary ids for.
    * @return All boundary IDs connected to elements in the give
    */
-  const std::set<BoundaryID> & getSubdomainBoundaryIds(SubdomainID subdomain_id) const;
+  const std::set<BoundaryID> & getSubdomainBoundaryIds(const SubdomainID subdomain_id) const;
```

Comment:
I probably didn't make it clear that I just wanted the `const` label on the parameter in the `.C` file as having it in the `.h` makes it harder to introduce some API changes (e.g. if we wanted to actually pass by non-const reference sometime in the future). But I don't imagine that we'll actually want to do that, and I'm not going to make you do more work if you don't want to when I motivated you to make the change

Reply:
Is it possible to have the function signature in C file different with the  one in the header? But I think keep it consistent is preferred. Even the argument is marked as non-const, I guess a const caller in a class can still call it with its member variables as the argument. So either way should be fine. But I think const indeed does not hurt. This is a big argument I do not want to get into though in this PR. This is good.

Reply:
Yes you can: https://stackoverflow.com/q/117293/4493669